pytorch_lightning/loops/epoch/__init__.py

```diff
@@ -14,4 +14,4 @@
 
 from pytorch_lightning.loops.epoch.evaluation_epoch_loop import EvaluationEpochLoop  # noqa: F401
 from pytorch_lightning.loops.epoch.prediction_epoch_loop import PredictionEpochLoop  # noqa: F401
-from pytorch_lightning.loops.epoch.training_epoch_loop import TrainingEpochLoop  # noqa: F401
+# from pytorch_lightning.loops.epoch.training_epoch_loop import TrainingEpochLoop  # noqa: F401
```

Comment:
@carmocca I'm trying to find an alternative here to resolve a cyclic import. 
seems pretty tricky to me

Reply:
Pushed 2 commits

1. grouping the imports to the largest level
2. Avoid the circular import by using `import * as *`

Tell me what you think of that

Reply:
I thought grouping to the largest level is one of the causes of the circular import

ok thanks